src/ripple/ledger/impl/View.cpp

```diff
@@ -391,7 +391,11 @@ transferRate (ReadView const& view,
     auto const sle = view.read(keylet::account(issuer));
 
     if (sle && sle->isFieldPresent (sfTransferRate))
+    {
+        if (sfTransferRate > 2 * QUALITY_ONE)
```

Comment:
This doesn't do what you expect it to. You're not retrieving the transfer rate entry.